### PR TITLE
【Hackathon No.21】add RFC for SoftMarginLoss

### PR DESCRIPTION
**Pull request by BlackWitty**

### CODE REVIEW

**Comment by dingjiaweiww** [comment]
请先签署cla噢~

**Comment by BlackWitty** [comment]
> 请先签署cla噢~

已经签署，辛苦CR

File: rfcs/APIs/20220412_api_design_for_soft_margin_loss.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+## 命名与参数设计
+共添加以下两个 API：
+```
+padde.nn.functional.soft_margin_loss(input Tensor[float64 or float32], target Tensor[float64 or float32],size_average [bool optional]默认为true,reduce [bool]作为参数，默认为true, reduction='mean', 'mean' 求平均,'sum'求和,'None',                                                negative, Tensor[float64 or float32],维度为[batch_size,dim]) -> Tensor
+ 
+ 
+ 
+paddle.nn.SoftMarginLoss(size_average=None, reduce=None, reduction='mean') -> Tensor
+该函数底层调用padde.nn.functional.soft_margin_loss
```
Comment (yingyibiao) [inline rfcs/APIs/20220412_api_design_for_soft_margin_loss.md:163]: 这里的size_average和reduce参数是否还有必要呢？
